auth_keycloak/models/res_users.py

```diff
@@ -0,0 +1,67 @@
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+from odoo import api, models, exceptions, _
+import logging
+import requests
+from ..exceptions import OAuthError
+
+logger = logging.getLogger(__name__)
+
+
+# NOTE: `sub` is THE user id in keycloak
+# https://www.keycloak.org/docs/latest/server_development/index.html#_action_token_anatomy  # noqa
+# and you cannot change it https://stackoverflow.com/questions/46529363
+
+
+class ResUsers(models.Model):
+    _inherit = 'res.users'
+
+    def _keycloak_validate(self, provider, access_token):
+        """Validate token against Keycloak."""
+        logger.debug('Calling: %s' % provider.validation_endpoint)
+        resp = requests.post(
+            provider.validation_endpoint,
+            data={'token': access_token},
```

Comment:
Have you faced a `400 bad request` motivated by Keycloak with _missing token_ using `token` keyword instead `access_token` ?